requirements.txt

```diff
@@ -1,4 +1,28 @@
-sphinx
-tox
-wheel
--e git+git@github.com:mesosphere/dcos-cli#egg=dcos
+alabaster==0.7.6
+Babel==2.1.1
+-e git+https://github.com/mesosphere/dcos-cli.git@372c3a5811ac537a104a5bf93bfa3fb32df44473#egg=dcos-0.2.0
```

Comment:
Since we now have the dcoscli in PyPI, we can just change this to `dcoscli==0.2.0``